Makefile

```diff
@@ -334,11 +341,34 @@ input_cleanup: ## deletes the secret/CM, used by the services as input
 	oc kustomize ${OUT}/${NAMESPACE}/input | oc delete --ignore-not-found=true -f -
 	${CLEANUP_DIR_CMD} ${OUT}/${NAMESPACE}/input
 
+##@ CRC BMO SETUP
+.PHONY: crc_bmo_setup
+crc_bmo_setup: export IRONIC_HOST_IP=${IRONIC_HOST} 
+crc_bmo_setup:
+	$(eval $(call vars,$@))
+	${CLEANUP_DIR_CMD} ${OPERATOR_BASE_DIR}/baremetal-operator
+	mkdir -p ${OPERATOR_BASE_DIR}
+	oc apply -f ${CERTMANAGER_URL}
+	pushd ${OPERATOR_BASE_DIR} && git clone ${GIT_CLONE_OPTS} $(if $(BMO_BRANCH),-b ${BMO_BRANCH}) ${BMO_REPO} "baremetal-operator" && popd
+	pushd ${OPERATOR_BASE_DIR}/baremetal-operator && sed -i 's/eth2/${PROVISIONING_INTERFACE}/g' ironic-deployment/default/ironic_bmo_configmap.env && popd
+	pushd ${OPERATOR_BASE_DIR}/baremetal-operator && make generate manifests && bash tools/deploy.sh -b -i && popd
+	## Hack to add required scc
+	oc adm policy add-scc-to-user privileged system:serviceaccount:baremetal-operator-system:baremetal-operator-controller-manager
+	oc adm policy add-scc-to-user privileged system:serviceaccount:baremetal-operator-system:default
+
+##@ CRC BMO CLEANUP
+.PHONY: crc_bmo_cleanup
+crc_bmo_cleanup:
+	oc kustomize ${OPERATOR_BASE_DIR}/baremetal-operator/ironic-deployment/default | oc delete --ignore-not-found=true -f -
+	oc kustomize ${OPERATOR_BASE_DIR}/baremetal-operator/config | oc delete --ignore-not-found=true -f -
+	${CLEANUP_DIR_CMD} ${OPERATOR_BASE_DIR}/baremetal-operator
+
 ##@ OPENSTACK
 .PHONY: openstack_prep
 openstack_prep: export IMAGE=${OPENSTACK_IMG}
 openstack_prep: $(if $(findstring true,$(NETWORK_ISOLATION)), nmstate nncp netattach metallb metallb_config) ## creates the files to install the operator using olm
 	$(eval $(call vars,$@,openstack))
+	if !(oc get crds | grep "baremetalhosts.metal3.io"); then make crc_bmo_setup; fi
```

Comment:
It looks like we install extra dependencies in this target, so this doesn't seem out-of-place.  I wonder, however, if there would be a way to get it into line 369 like we do with the other dependencies.

Reply:
done